addons/snackpacks/CfgWeapons.inc

```diff
@@ -0,0 +1,118 @@
+        picture = QPATHTOF(data\ui\snack_ui_shorks.paa);
+    };
+    class SWST_TigerThymes : SWST_Base_Snack {
+        ITEM_META(2);
+
+        displayName = "Tiger Thymes";
+        descriptionShort = "A pouch of savory jerky snacks. Herbacious flavor.<br/>May cause cheerfulness even in the face of hopeless odds.";
+
+        hiddenSelections[] = {"camo"};
+        hiddenSelectionsTextures[] = {QPATHTOF(data\snackbox\snackbox_tiger_co.paa)};
+        picture = QPATHTOF(data\ui\snack_ui_tiger.paa);
+    };
+    class SWST_SonicBooms : SWST_Base_Snack {
+        ITEM_META(2);
+
+        displayName = "Sonic Booms";
+        descriptionShort = "Packet of mini chocolate chip cookies.<br/>Nobody knows why there's a hedgehog on the package.";
+
+        hiddenSelections[] = {"camo"};
+        hiddenSelectionsTextures[] = {QPATHTOF(data\snackbox\snackbox_blue_co.paa)};
+        picture = QPATHTOF(data\ui\snack_ui_blue.paa);
+    };
+    class SWST_EggCandy : SWST_Base_Snack {
+        ITEM_META(2);
+
+        displayName = "Egg's Sour Rockets";
+        descriptionShort = "Pack of extremely sour candies.<br/>May cause cravings for breakfast food.";
+
+        hiddenSelections[] = {"camo"};
+        hiddenSelectionsTextures[] = {QPATHTOF(data\snackbox\snackbox_blue_co.paa)};
+        picture = QPATHTOF(data\ui\snack_ui_blue.paa);
+    };
+    class SWST_BunnySnack : SWST_Base_Snack {
+        ITEM_META(2);
+
+        displayName = "Bloody Bunny Treats";
+        descriptionShort = "Packet of delicious jerky.<br/>Surely these aren't actually the hearts of Rabbit's enemies...";
+
+        hiddenSelections[] = {"camo"};
+        hiddenSelectionsTextures[] = {QPATHTOF(data\snackbox\snackbox_blue_co.paa)};
+        picture = QPATHTOF(data\ui\snack_ui_blue.paa);
+    };
+    class SWST_OrangeSlices : SWST_Base_Snack {
+        ITEM_META(2);
+
+        displayName = "Orange Slices";
+        descriptionShort = "Gummy candy in the shape and flavor of orange slices.<br/>Reportedly, they tempt one towards kleptomania.";
+
+        hiddenSelections[] = {"camo"};
+        hiddenSelectionsTextures[] = {QPATHTOF(data\snackbox\snackbox_blue_co.paa)};
+        picture = QPATHTOF(data\ui\snack_ui_blue.paa);
+    };
+    class SWST_SonicBooms : SWST_Base_Snack {
+        ITEM_META(2);
+
+        displayName = "Sonic Booms";
+        descriptionShort = "Packet of mini chocolate chip cookies.<br/>Nobody knows why there's a hedgehog on the package.";
+
+        hiddenSelections[] = {"camo"};
+        hiddenSelectionsTextures[] = {QPATHTOF(data\snackbox\snackbox_blue_co.paa)};
+        picture = QPATHTOF(data\ui\snack_ui_blue.paa);
+    };
+    class SWST_AloeWater : SWST_Base_Drink {
+        ITEM_META(2);
+
+        displayName = "Aloe Water";
+        descriptionShort = "Smooth Jazz's Smooth Aloe Water.<br/>Tastes like placeholder model.";
+        // Placeholder item - update with textures and such later.
```

Comment:
If this isn't ready yet, just make it `SCOPE(0)` for now and come back around to it.

Reply:
It's drinkable, it just doesn't have a good worldmodel yet.